snaps_k8s/provision/ansible_configuration.py

```diff
@@ -902,3 +929,189 @@ def delete_weave_interface(k8s_conf):
         }
         ansible_utils.apply_playbook(consts.K8_DELETE_WEAVE_INTERFACE,
                                      variables=pb_vars)
+def __get_loadbalancer_dict(config):
```

Comment:
Please move me to config_utils.py or change the get_ha_config() function to match as other calls would not  be really working anyhow